app/components/sendverificationbanner/sendverificationbanner.js

```diff
@@ -0,0 +1,64 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { translate } from 'react-i18next';
+
+import personUtils from '../../core/personutils';
+
+const SendVerificationBanner = (props) => {
+  const {
+    patient,
+    trackMetric,
+    resendVerification,
+    t,
+    resendEmailVerificationProgress: working,
```

Comment:
Wondering if this would be better as `resendingEmailVerification` or `resendEmailVerificationInProgress`
The name of this prop doesn't suggest to me that it would be a Boolean.  

No great concern here, and if you feel otherwise, just merge it as-is.

Reply:
Yeah, going with the clearer and more verbose. Same as the unit test below.